widgetastic_manageiq/__init__.py

```diff
@@ -2468,7 +2468,7 @@ def __repr__(self):
 
     def __init__(self, parent, locator=None, logger=None):
         super(TimelinesChart, self).__init__(parent=parent, logger=logger)
-        self.locator = locator or '//div[contains(@class, "timeline-container")]'
+        self.locator = locator or '//div[@id="tl_div"]'
```

Comment:
QUESTION: are you sure this won't break tests against 5.8 ?

Reply:
I tried it works